carbon-design-system/carbon
# feat(react): add initial content switcher refactor

Initial implementation of #9519 

This PR adds in a base implementation and API for the new `ContentSwitcher` component

#### Changelog

**New**

- Add initial implementation of `ContentSwitcher` for v11

**Changed**

**Removed**

#### Testing / Reviewing

- Pull down the PR locally and run the storybook in `packages/react`
- Comment the line in `next/ContentSwitcher-story.js` that says `includeStories: []`
- Try out the story and make sure that the ContentSwitcher works with both keyboard and mouse for switching the panels
